collectors/python.d.plugin/portcheck/README.md

```diff
@@ -1,6 +1,6 @@
-# portcheck
+# PortCheck monitoring with Netdata
```

Comment:
@ilyam8 What would you recommend here?

Reply:
i would recomment follow `Monitors` column from https://docs.netdata.cloud/collectors/collectors/

Reply:
portcheck monitors any tcp endpoint

Reply:
Okay, so **# TCP endpoint monitoring with Netdata**

Reply:
@ilyam8 I'll follow the Monitors column you suggested.  Thanks :)

Reply:
thank you @shortpatti 👍